packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -489,7 +490,7 @@ const VirtualListBaseFactory = (type) => {
 							const {dataSize, focusableScrollbar} = this.props;
 							const column = index % dimensionToExtent;
 							const row = (index - column) % dataSize / dimensionToExtent;
-							const isLeaving = directions.up && row === 0 ||
+							isLeaving = directions.up && row === 0 ||
 								directions.down && row === Math.floor((dataSize - 1) % dataSize / dimensionToExtent) ||
 								directions.left && column === 0 ||
 								directions.right && !focusableScrollbar && column === dimensionToExtent - 1;
```

Comment:
Even though `focusableScrollbar` is equal to `true`, Spotlight leaves from VirtualList if `verticalScrollbar` prop and `horizontalScrollbar` prop in `Scrollable` are `'hidden'` because there is no scrollbars in the list.

Reply:
Example
```
storiesOf('Moonstone', module)
	.add(
		'VirtualList.VirtualGridList',
		() => (
			<div style={{width: '100%', height: '100%'}}>
				<VirtualGridList
					dataSize={5}
					direction="horizontal"
					focusableScrollbar="true"
					itemRenderer={renderItem}
					itemSize={{
						minWidth: 180,
						minHeight: 50
					}}
					spacing={0}
					verticalScrollbar="hidden"
					horizontalScrollbar="hidden"
					style={{width: '100%', height: '50%'}}
				/>
				<GridListImageItem
					caption="test"
					subCaption="test"
					style={{position: 'static'}}
				/>
			</div>
		),
		{
			info: {
				text: 'Basic usage of VirtualGridList'
			}
		}
	);
```

Reply:
Thanks, @ybsung - that's an excellent observation! It appears we've been overlooking whether or not the user explicitly hides the scrollbar :(
I've updated the PR to consider these scenarios. Can you review it again?